packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "salesforcedx-vscode-lwc-next",
+  "displayName": "LWC Code Editor for Visual Studio Code",
+  "description": "Provides code-editing features for LWC",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/VSCodeLWC.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "42.2.0",
+  "publisher": "salesforce",
+  "preview": true,
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.17.0"
+  },
+  "categories": ["Languages"],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "42.2.0",
+    "eslint": "4.16.0",
+    "eslint-plugin-lwc": "0.3.2",
+    "lwc-language-server": "0.16.4",
```

Comment:
@rsalvador - This is where we will use the 1.x.y series of the lwc language server.